### ISSUE TITLE
Question? <x-slide></x-slide> internal scroll

### ISSUE THREAD

**Issue by marti1125** [mozbrick/brick#47]
Could I add internal scroll using overflow: scroll ?
and also I have tabs with different height.

**Comment by ldoubleuz** [comment]
Hi,

For your question regarding scroll, yes, you could do that, as x-slides can be styled just like any other native dom element.

As for your question regarding tabs, I'm not sure what you're trying to ask. Could you rephrase the question?

**Comment by marti1125** [comment]
is for tab containers

**Comment by ldoubleuz** [comment]
Er, sorry, I'm afraid I don't understand what you are asking. What about the tab containers?

**Comment by marti1125** [comment]
I am using this example - With <x-slidebox> (also dynamically assigned tab targets) http://mozilla.github.io/brick/demos/tabbar/index.html

![scroll](https://f.cloud.github.com/assets/223240/1064286/c4587422-1322-11e3-9349-aa90826d10e4.png)

in this slide I am loading all content via backbonejs

<x-slide><div id="tweets"></div></x-slide>

![scroll2](https://f.cloud.github.com/assets/223240/1064302/6e9bcf7e-1323-11e3-85f2-2722dd668998.png)

**Comment by ldoubleuz** [comment]
Okay, and what would the issue be?

**Comment by marti1125** [comment]
Sorry :(
I correct my mistake 

```
#tweets {
width:320px;
height:480px;
overflow: scroll;
}
```

is possible you add automatically scroll in <x-slide></x-slide> when the content is big

**Comment by ldoubleuz** [comment]
Sounds like you want the behavior of `overflow: auto`, which you can apply to the slide like any other element.

**Comment by marti1125** [comment]
yes :) thanks for you help me
